src/parser/detectHandler.js

```diff
@@ -4,13 +4,20 @@ const handlers = require('../handlers/index')
 const noop = () => undefined
 
 module.exports = function detectHandler(message) {
-  const requestedHandler = message.split(' ').shift()
+  const requestedNamespace = message.split(' ')[0]
+  let requestedHandler = message.split(' ')[1]
```

Comment:
Using array destructuring we can do this in a more readable way:
`const [requestedNamespace, requestedHandler] = message.split(' ')`